src/com/owncloud/android/ui/dialog/RemoveFileDialogFragment.java

```diff
@@ -93,11 +95,16 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
     public void onConfirmation(String callerTag) {
         ComponentsGetter cg = (ComponentsGetter)getActivity();
         FileDataStorageManager storageManager = cg.getStorageManager();
-        if (storageManager.getFileById(mTargetFile.getFileId()) != null) {
+        OCFile ocFile = storageManager.getFileById(mTargetFile.getFileId());
+        if (ocFile != null) {
             cg.getFileOperationsHelper().removeFile(mTargetFile, false);
+            Intent i = new Intent(getActivity(), MediaService.class);
+            i.putExtra(MediaService.EXTRA_FILE, ocFile);
+            i.setAction(MediaService.ACTION_STOP_FILE);
+            getActivity().startService(i);
```

Comment:
It would be great if you set this behaviour also for a local remove, since it makes sense that if you want to remove the file locally, you don't want to keep it in the buffer of the media service.

P.D: The local remove listener is currently set in the negative button (which listener mehotd is onCancel). Extract to a new method to reuse the MediaService call.